packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -480,6 +480,31 @@ export const PostSharedWithUserNotification = registerNotificationType({
   }
 });
 
+export const PostAddedAsCoauthorNotification = registerNotificationType({
+  name: "addedAsCoauthor",
+  userSettingField: "notificationAddedAsCoauthor",
+  allowedChannels: ["onsite", "email", "both"],
+  async getMessage({documentType, documentId}: GetMessageProps) {
+    let document = await getDocument(documentType, documentId) as DbPost;
+    const name = await postGetAuthorName(document);
+    const postOrDialogue = document.collabEditorDialogue ? 'dialogue' : 'post';
+    return `${name} added you as a coauthor to the ${postOrDialogue} "${document.title}"`;
+  },
+  getIcon() {
+    return <Components.ForumIcon icon="Bell" style={iconStyles} />
```

Comment:
nbd but we use this other icon for when someone requests you be added as a coauthor, you might want to use that icon here (or maybe the post icon? since the notification links to a post)

<img width="278" alt="Screen Shot 2023-10-31 at 2 12 30 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/9dada329-be3a-4ffe-85cd-2d129bc5ea8c">

Reply:
Oh, yeah, makes sense.